pkg/util/env/env.go

```diff
@@ -24,7 +24,10 @@ import (
 )
 
 const (
-	NodeNameEnvKey        = "NODE_NAME"
+	NodeNameEnvKey = "NODE_NAME"
+	// ExternalNodeEnvKey is an environment variable used in ExternalNode feature to tell the expected name with which
```

Comment:
to populate the name of the VM Node on which Antrea Agent is running

Reply:
done.